### DISCUSSION TITLE
CI/CD for Detox

### DISCUSSION THREAD

**Discussion by arunav1101** [wix/Detox#3201] [Q&A]
Can we run the detox on bitbucket?

I saw the options with Travis CI , bitrise and gitlab but not with bitbucket.

**Comment by yuriy1715** [comment]
The same problem, Can someone give a feedback

**Comment by d4vidi** [comment]
I can't give either a _yes_ or a _no_ here, because we (maintainers) have never tried running on Bitbucket ourselves.

In order to help nevertheless I would advise to equivalently check whether on Bitbucket you are capable of setting up build jobs over [proper execution environment](shttps://wix.github.io/Detox/docs/introduction/getting-started#step-1-environment-setup): Largely, a MacOS that can run simulators, for iOS, or a linux system that can run an Android emulator and can provide access to an Android SDK, etc, for Android.
